site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -164,6 +167,9 @@ image::core-concepts/tenant-configuration-email-verification-settings.png[Tenant
 [field]#Verify email# [optional]#Optional#::
 When enabled, users will be required to verify their email address.
 
+[field]#Allow implicit verification# [optional]#Optional#::
```

Comment:
Since

Reply:
Not sure if we usually add a default to these? This is defaulted to `true` for backwards compatibility.